docs/source/conf.py

```diff
@@ -60,7 +59,7 @@
 
 # General information about the project.
 project = u'Pyro'
-copyright = u'2017, Uber Technologies Inc'
+copyright = u'2018, Uber Technologies Inc'
```

Comment:
Copyright years should be inclusive: 2017-2018.

Reply:
im pretty sure they forgot to do this.. otherwise theyre usually inclusive `2017-`. in general you need an included year for every year the software is updated
http://www.gnu.org/licenses/gpl-howto.html

Reply:
@fritzo i did that in the license but [tf overwrote theirs](https://github.com/tensorflow/tensorflow/commit/6b6d843ccab78f9f91c3b98a43ca09ffecad4747)

Reply:
tf also forgot to update `2017` at the bottom of the same page. :)

Reply:
@neerajprad tf was released in 2015, im pretty sure thats true... ill make it inclusive though i think that should make everyone happy